README.md

```diff
@@ -0,0 +1,19 @@
+# ODK Skunkworks
+
+[![Code of Conduct](https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-blue.svg?style=flat)](https://github.com/opendatakit/governance/blob/master/CODE-OF-CONDUCT.md)
+
+Open Data Kit (ODK) is a collection of free and open-source software tools that help people capture mission-critical data in real-time through mobile technology. ODK Skunkworks is a listing of projects that are experimental and not yet released for broad deployment. Learn more about skunkworks projects [on Wikipedia](https://en.wikipedia.org/wiki/Skunkworks_project).
+
+## Learn more about ODK
+* ODK website: [https://opendatakit.org](https://opendatakit.org)
+* ODK usage documentation: [https://docs.opendatakit.org/](https://docs.opendatakit.org/)
+* ODK forum: [https://forum.opendatakit.org](https://forum.opendatakit.org)
+* ODK developer Slack chat: [http://slack.opendatakit.org](http://slack.opendatakit.org) 
+* ODK developer Slack archive: [https://opendatakit.slackarchive.io](https://opendatakit.slackarchive.io) 
+
+## Current ODK skunkworks projects
+### Supervisor app - Device to Device Transfer Android Application 
+Lead: Lakshya Gupta ([@lakshyagupta21](https://github.com/lakshyagupta21)), GSoC 2018 intern
+
+## Adding new skunkworks projects
```

Comment:
How specifically does one propose a skunkworks project? On the forum in a TSC meeting agenda?